tests/integration/balance/balance.spec.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  BrowserDriver,
+  createTestSelector,
+  selectTestNet,
+  setupBrowser
+} from '../utils';
+import { WalletPage } from '../../page-objects/wallet.page';
+import { ScreenPaths } from '@common/types';
+import {BalanceSelectors} from "@tests/integration/balance.selectors";
```

Comment:
This is against our prettier config. Make sure your IDE has the prettier plugin installed

Reply:
Ok I am using webstorm let me add it

Reply:
So its just a spacing issue I am guessing that the only thing that updated I can push this up,